frontend/src/components/workspace/CreateApplicationButton.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { startApplication } from "src/services/fetch/fetchers/clientApplicationFetcher";
+
+import { useRouter } from "next/navigation";
+import { useCallback, useState } from "react";
+import { Button } from "@trussworks/react-uswds";
+
+const CreateApplicatinButton = () => {
+  const router = useRouter();
+  const [loading, setLoading] = useState<boolean>();
+
+  const handleSubmit = useCallback(() => {
+    setLoading(true);
+    startApplication("insecuretoken")
```

Comment:
can't we use the token from "useUser"?

Reply:
We aren't doing logged in users so it is out of scope.

Reply:
I'm confused - don't we want to limit this to logged in users, or is there an issue with that being too complex of a use case for demo purposes?

Reply:
Logged in is out of scope